packages/pyright-internal/src/tests/docStringConversion.test.ts

```diff
@@ -746,31 +746,6 @@ Returns:
     _testConvertToMarkdown(docstring, markdown);
 });
 
-test('GoogleWithInvalidTypes', () => {
```

Comment:
Now that are regex is very relaxed we are keeping the line break in this example. so its no longer a failure case.